framework/src/utils/PetscSupport.C

```diff
@@ -620,8 +620,9 @@ storePetscOptions(FEProblemBase & fe_problem, const InputParameters & params)
      * which happens before the parser is even created.  We'll throw an error if somebody attempts
      * to add this option later.
      */
-    if (option == "-log_summary")
-      mooseError("The PETSc option \"-log_summary\" can only be used on the command line.  Please "
+    if (option == "-log_summary" || option == "-log_view")
+      mooseError("The PETSc option \"-log_summary\" or \"-log_view\" can only be used on the "
+                 "command line.  Please "
```

Comment:
Why is this?

Reply:
We want to stop users from using -log_summary or -log_view in input file. Because that won't work. PETSc needs to parse this option really early to get log information correct

Reply:
Got it